app/models/snack.rb

```diff
@@ -0,0 +1,3 @@
+class Snack < ApplicationRecord
+
+end 
```

Comment:
Snack will also need to have a relationship set up at the model level. Snacks will have many machines through machine_snacks